moirae/estimators/online/utils.py

```diff
@@ -2,6 +2,8 @@
 import numpy as np
 
 
+# TODO (wardlt): Why do we have two
+
```

Comment:
Can probably be merged in a single function, I just thought this was more streamlined: one function to find the nearest positive semi-definite matrix, and one for figuring out if we need to use the algorithm or not. But, tbh, I can't foresee a case where they would not be used in tandem (one calls the other afterall...), so, if you feel like they can be implemented together, you can go ahead on that

Reply:
Sorry, didn't realize that comment made it in there. I wrote it while having a brain fart and not realizing "enforce" and "ensure" are different words.

Let's keep it as-is.